src/components/DecisionDetail/VoteCasted.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import { GU, IconCheck, RADIUS, textStyle, useTheme } from '@1hive/1hive-ui'
-import useExtendedVoteData from '../../hooks/useExtendedVoteData'
 import { useAppState } from '../../providers/AppState'
+import { formatTokenAmount } from '../../lib/token-utils'
+import { getAccountCastStake } from '../../lib/vote-utils'
 import { VOTE_YEA } from '../../constants'
 
-function VoteCasted({ vote }) {
-  const { connectedAccountVote } = vote
-  const { userBalance } = useExtendedVoteData(vote)
+function VoteCasted({ account, accountVote, vote }) {
   const { config } = useAppState()
+  const accountStake = getAccountCastStake(vote, account)
```

Comment:
ah getting it from the hook was not working?

Reply:
As discussed previously, the user's snapshot is not always correct as the users voting weight as they could have transferred some amount before voting